meos/include/point/tpoint_rtree.h

```diff
@@ -74,14 +74,23 @@ typedef struct RTreeNode{
   STBox boxes[MAXITEMS];
 } RTreeNode;
 
-
-typedef struct RTreeMetadata {
+/**
+ * Rtree in memory index basic structure.
+ *
+ * It works based on STBox. The spliting criteria is based on the largest axis.
+ * The inserting criteria is based on least enlarging square.
+ *
+ * The get axis function makes it ease to implement with X,Y,Z and time or any
+ * combination that you may want.
+ */
+struct RTree {
```

Comment:
I see. Didn't thought about using this! thanks!

LGTM we can go ahead and merge it!